features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Base/Header/EnvironmentMenu.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from "react";
+import Button from 'material-ui/Button';
+import Menu, {MenuItem} from 'material-ui/Menu';
+
+class EnvironmentMenu extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            openEnvironmentMenu: false
+        };
+    }
+
+    handleClickEnvironmentMenu = event => {
```

Comment:
Shall we avoid using ES7 class properties, As stated in react [documentation](https://reactjs.org/docs/handling-events.html) this is an experimental feature. If we use this syntax we have to use this babel [plugin](https://babeljs.io/docs/plugins/transform-class-properties/) which we don't use.

Reply:
Done with https://github.com/wso2/carbon-apimgt/pull/4890/commits/57e9929cdc68c5b4447681c94a793b86b0390380